src/Storages/System/StorageSystemZooKeeperConnection.cpp

```diff
@@ -67,13 +67,13 @@ void StorageSystemZooKeeperConnection::fillData(MutableColumns & res_columns, Co
             UInt16 port = static_cast<UInt16>(Poco::NumberParser::parseUnsigned(host_port.substr(offset + 1)));
 
             UInt32 uptime = zookeeper->getSessionUptime();
-            time_t time = timeInSeconds(std::chrono::system_clock::now()) - uptime;
+            time_t connected_time = time(nullptr) - uptime;
```

Comment:
I wonder why don't we get the start time from the watch itself. `Stopwatch` has a `getStart()` function that's exact and that allows us to ignore possible  time adjustments.

Reply:
It does not correspond to the absolute time:

```
CLOCK_MONOTONIC
              A nonsettable system-wide clock that represents monotonic time since—as described by POSIX—"some unspecified point in the past".  On Linux, that point corresponds to the number of seconds  that  the  system  has
              been running since it was booted.
```